akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -212,7 +212,7 @@ object Source {
    * receive new tick elements as soon as it has requested more elements.
    */
   def apply[T](initialDelay: FiniteDuration, interval: FiniteDuration, tick: T): Source[T, Cancellable] =
-    fromGraph(new TickSource[T](initialDelay, interval, tick))
+    fromGraph(new TickSource[T](initialDelay, interval, tick).withAttributes(DefaultAttributes.iterableSource))
```

Comment:
Yes, this apply should be renamed to tickSource. Please add the change to the list https://github.com/akka/akka/issues/18737 so we don't forget to add it to the migration guide.